src/graph_notebook/magics/graph_magic.py

```diff
@@ -87,10 +97,12 @@ def __init__(self, shell):
         super(Graph, self).__init__(shell)
 
         try:
-            self.graph_notebook_config = get_config()
+            self.config_location = os.getenv('GRAPH_NOTEBOOK_CONFIG', DEFAULT_CONFIG_LOCATION)
```

Comment:
Is this something we will expose to users?

Reply:
It is not documented yet, but we should add this to our README